### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #893 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#431]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 1.29 KB...]
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.5.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: https://jitpack.io/net/minidev/json-smart/maven-metadata.xml
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.6.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 75 source files to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/ProfileImageService.java:[60,79] cannot find symbol
  symbol:   class RequestHeader
  location: class smartcity.accessibility.services.ProfileImageService
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.473 s
[INFO] Finished at: 2017-06-13T17:30:48+00:00
[INFO] Final Memory: 32M/212M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/ProfileImageService.java:[60,79] cannot find symbol
[ERROR] symbol:   class RequestHeader
[ERROR] location: class smartcity.accessibility.services.ProfileImageService
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] b5c16d80506ae18b948fb62d00dfa239eb94a683 - move token to header in profileImgService

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/893/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
